test/e2e/volume_provisioning.go

```diff
@@ -109,16 +109,16 @@ var _ = KubeDescribe("Dynamic provisioning", func() {
 			// 10 minutes here. There is no way how to see if kubelet is
 			// finished with cleaning volumes. A small sleep here actually
 			// speeds up the test!
-			// One minute should be enough to clean up the pods properly.
-			// Detaching e.g. a Cinder volume takes some time.
+			// Three minutes should be enough to clean up the pods properly.
+			// We've seen GCE PD detach to take more than 1 minute.
 			By("Sleeping to let kubelet destroy all pods")
-			time.Sleep(time.Minute)
+			time.Sleep(3 * time.Minute)
```

Comment:
Well, I _could_, but as you can notice, AWS and Cinder is not implemented there.  Common cloud volume itnerface as suggested in https://github.com/kubernetes/kubernetes/issues/21984 would help a lot (with `IsDiskAttached()` method).

I'd propose fixing the flake now and polishing the timers later.